ess/src/API/EMBC.ESS/Resources/Cases/Evacuations/EvacuationRepository.cs

```diff
@@ -253,7 +253,18 @@ private static async Task ParallelLoadEvacuationFileAsync(EssContext ctx, era_ev
             loadTasks.Add(Task.Run(async () => await ctx.LoadPropertyAsync(file, nameof(era_evacuationfile.era_era_evacuationfile_era_animal_ESSFileid))));
             loadTasks.Add(Task.Run(async () => await ctx.LoadPropertyAsync(file, nameof(era_evacuationfile.era_era_evacuationfile_era_essfilenote_ESSFileID))));
             loadTasks.Add(Task.Run(async () => await ctx.LoadPropertyAsync(file, nameof(era_evacuationfile.era_TaskId))));
-            loadTasks.Add(Task.Run(async () => await ctx.LoadPropertyAsync(file, nameof(era_evacuationfile.era_era_evacuationfile_era_evacueesupport_ESSFileId))));
+
+            loadTasks.Add(Task.Run(async () =>
+            {
+                await ctx.LoadPropertyAsync(file, nameof(era_evacuationfile.era_era_evacuationfile_era_evacueesupport_ESSFileId));
+                foreach (var support in file.era_era_evacuationfile_era_evacueesupport_ESSFileId)
+                {
+                    support.statuscode = (int?)(support.era_validto < DateTime.UtcNow ? SupportStatus.Expired : SupportStatus.Active);
```

Comment:
the status should be set in Dynamics and we should just read it and map it in the mapper